SQL/Create_Table_PetActivity.sql

```diff
@@ -0,0 +1,21 @@
+-- Drop the table if it already exists
+IF OBJECT_ID('PetGame.PetActivity', 'U') IS NOT NULL
+DROP TABLE PetGame.PetActivity
+GO
+-- Create the table in the specified schema
+CREATE TABLE PetGame.PetActivity
+(
+    -- primary key for for table
+    ActivityId BIGINT IDENTITY(1,1) NOT NULL PRIMARY KEY, 
+    
+    -- foreign key from PET.PetId
+    PetId BIGINT NOT NULL FOREIGN KEY REFERENCES PET(PetId),
```

Comment:
`PET` table is not uppercase, should be `Pet`.
`NOT NULL` is redundant
```suggestion
    PetId BIGINT FOREIGN KEY REFERENCES [dbo].[Pet](PetId),
```